packages/client-adapters/tests/transformers/alpheios-lexicon-transformer.test.js

```diff
@@ -153,4 +154,22 @@ describe('transform-adapter.test.js', () => {
     expect(homonym.lexemes[0].lemma.word).toEqual('hdwdb')
     expect(homonym.lexemes[0].inflections.length).toEqual(1)
   })
+
+  it('2 TransformAdapter - makes sure Lemmas are strings', async () => {

```

Comment:
I'm thinking maybe we should also add a test case when `$` is not defined? What do you think?

Reply:
yes good idea.